infrastructure/src/main/java/org/corfudb/infrastructure/LocalMonitoringService.java

```diff
@@ -0,0 +1,138 @@
+package org.corfudb.infrastructure;
+
+import static org.corfudb.util.LambdaUtils.runSansThrow;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+
+import java.time.Duration;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import org.corfudb.infrastructure.management.ClusterStateContext;
+import org.corfudb.protocols.wireprotocol.SequencerMetrics;
+import org.corfudb.protocols.wireprotocol.SequencerMetrics.SequencerStatus;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.exceptions.ServerNotReadyException;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.util.CFUtils;
+import org.corfudb.util.concurrent.SingletonResource;
+
+/**
+ * Local Metrics Polling Service polls the local server state.
+ * Created by zlokhandwala on 11/2/18.
+ */
+@Slf4j
+class LocalMonitoringService implements IService {
+
+    private final ServerContext serverContext;
+    private final SingletonResource<CorfuRuntime> runtimeSingletonResource;
+    private final ClusterStateContext clusterStateContext;
+
+    //  Locally collected server metrics polling interval.
+    private static final Duration METRICS_POLL_INTERVAL = Duration.ofMillis(3000);
+
+    private final ScheduledExecutorService pollingService;
+
+    LocalMonitoringService(@NonNull ServerContext serverContext,
+                           @NonNull SingletonResource<CorfuRuntime> runtimeSingletonResource,
+                           @NonNull ClusterStateContext clusterStateContext) {
```

Comment:
Please pass `LayoutView` instead of `CorfuRuntime`

Reply:
https://github.com/CorfuDB/CorfuDB/issues/1564